src/Symfony/Component/Notifier/Bridge/Lox24/Type.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Symfony\Component\Notifier\Bridge\Lox24;
+
+enum Type: string
```

Comment:
Type is a bit generic no?

Reply:
I thought the same, but the namespace seems enough to me

Reply:
Well not that much ... type of what ? Recipient ? Payment ? Security ? I'm sure we can find a more precise word (that could also help DX, beeing maybe more self-explanatory)

(after nothing wrong here, and i'm just suggesting of course)

Reply:
@smnandre MessageType?

Reply:
No @OskarStark is right, it's isolated and cannot really be better in the end :| Forget this :)

Reply:
@OskarStark some news? when it will be merged?